examples/ShowPlayer.cpp

```diff
@@ -216,15 +216,18 @@ ShowLoader::State ShowPlayer::SeekTo(uint64_t seek_time) {
       break;
     }
   }
+  uint64_t timeout = playhead_time - seek_time;
   m_playback_pos = playhead_time;
+  m_clock.CurrentTime(&m_start_ts);
```

Comment:
You should update your branch against master and then switch to CurrentMonotonicTime as part of https://github.com/OpenLightingProject/ola/pull/1673